app/controllers/miq_policy_controller.rb

```diff
@@ -284,8 +233,38 @@ def fetch_log
                        :message => "Policy log downloaded")
   end
 
+  # need to load toolbar here, because rsop screen code is in the same controller.
+  def show_list
+    super
+    @center_toolbar = "miq_policies"
+  end
+
+  # Get information for a policy
+  def show
+    super
+    @center_toolbar = "miq_policy"
+    @record = @policy = MiqPolicy.find_by(:id => params[:id])
+    @policy_conditions = @policy.conditions
+    @policy_events = @policy.miq_event_definitions
+    @expression_table = @policy.expression.kind_of?(MiqExpression) ? exp_build_table(@policy.expression.exp) : nil
+    @policy_profiles = @policy.memberof.sort_by { |pp| pp.description.downcase }
+  end
```

Comment:
This seems also weird...

Reply:
> This seems also weird...

Addressed this issue by removing unnecessary set up on instance variable in this method